src/LogBackend.cxx

```diff
@@ -62,28 +71,56 @@ SetLogThreshold(LogLevel _threshold) noexcept
 	log_threshold = _threshold;
 }
 
+template <typename DurationType>
+inline constexpr const char *date_format_of = "";
```

Comment:
`inline` is redundant here, but why is it not `static`? Without `static`, it's being exported and may conflict with symbols from other compilation units.